service/k8sclient/k8client_test.go

```diff
@@ -126,12 +146,31 @@ func TestK8Client(t *testing.T) {
 		})
 		require.NoError(t, err)
 
-		l.Info("PSMDB Cluster is created")
-
 		assertListPSMDBCluster(t, ctx, client, name, func(cluster *PSMDBCluster) bool {
 			return cluster != nil && cluster.State == ClusterStateReady
 		})
 
+		getCluster := func(name string) (cluster PSMDBCluster) {
```

Comment:
Let's move this function out of current test and reuse it in assertListxxxCluster functions

Reply:
Ok, I moved it and reused it.

In some future sprint, we should refactor at least tests so we don't duplicate code so much.